### PR TITLE
[m] make API_KEY configurable

### PR DESCRIPTION
**Pull request by subhankarb**
Previously we hardcoded the API_KEY in BaseConfig. But It supposed
to be configurable.
 * Changed API_KEY name to JWT_SEED, as it is used to create jwt token.
 * Made API_KEY configurable for Staging and dev environment.

Resolves : #206

### CODE REVIEW

File: app/config.py [hunk 1]
```diff
@@ -77,5 +80,11 @@ class DevelopmentConfig(BaseConfig):
 
 
 class StageConfig(DevelopmentConfig):
+
+    required_config = ['S3_BUCKET_NAME', 'JWT_SEED',
```
Comment (zelima) [inline app/config.py]: what is this for?

Comment (zelima) [inline app/config.py]: what is this for?

Comment (subhankarb) [inline app/config.py]: `API_KEY` (renamed to `JWT_SEED`) supposed to be supplied from external file or environment variable for staging and prod configuration.

Comment (zelima) [inline app/config.py]: Yes, just think it's defined and never used

**Pull request merged**